students/charlie_england/session04/mailroom_part2.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+#refactor to have the donors dict be 2 lists
+import sys
+# Donors dict Name:[[list of donations],[sum donations, num donations, average donations]]
+donors = {"Chris Christly": [[1000.21, 250.80],[]], "Bob Barley": [[800.33],[]], "Nick Nilly": [[
```

Comment:
Be careful here, you don't actually want to have a list of lists for the donations. You can just have something like:
{{"Chris Christly": [1000.21, 250.80], "Bob Barley": [800.33], ... ] 

Otherwise, you will run into issues when trying to compute stats. When I add more donations, it doesn't update stats properly and I believe this is the issue.